src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -79,6 +79,9 @@ message ConfigProto {
 
   // The language of the generated code.
   string language = 5;
+  
+  // The configuration to create the providers.
+  GeneratorProto generator = 6;
```

Comment:
Discovery still uses them. We can remove them after Discovery is refactored to this new pattern.